### ISSUE TITLE
Bottom padding alignment issue (from Joe)

### ISSUE THREAD

**Issue by romichg** [TeamMentor/TM_4_0_Design#1021]
I noticed the bottom padding doesn’t line up, I added the lines here to make it more obvious.
![screenshot 2015-09-17 13 29 34](https://cloud.githubusercontent.com/assets/1326120/9949466/013b7778-5d7b-11e5-80eb-14b6c9f1540f.PNG)

**Comment by luchia** [comment]
Yup, for some reason @michaelhidalgo height code isn't working on the middle column anymore.

**Comment by romichg** [comment]
so is this your or @michaelhidalgo fix?

**Comment by michaelhidalgo** [comment]
Hi @luchia , I don't think it is related with height code. I just hide or show the scroll dynamically. Note that three columns are not aligned (left hand side navigation, middle navigation and right hand side navigation).

**Comment by luchia** [comment]
Okay, but that needs to be fixed because it's now in the beta. The bottoms will line up when the .scrolling-results works properly.

![screen shot 2015-09-23 at 6 21 49 pm](https://cloud.githubusercontent.com/assets/4547737/10040542/faeda40a-621f-11e5-9b3b-d35268a1e1b4.png)

If you go the index page, the left and right hand columns do line up properly., the middle doesn't. The only reason the left-hand column won't line up properly is if the user navigates to another page where the index doesn't contain as many directories, like below -

![screen shot 2015-09-23 at 6 24 02 pm](https://cloud.githubusercontent.com/assets/4547737/10040578/4999e258-6220-11e5-95d1-ad25ac340422.png)

**Comment by romichg** [comment]
It seems like the issue happens when y0u make the original size of the browser smaller.  Also seems like the bottom controls are scrolled below the screen. Here is how it looks like:
![issue_1021](https://cloud.githubusercontent.com/assets/1326120/10272012/557d0742-6ae7-11e5-9a5d-942ebd40a484.gif)
